pyccel/parser/semantic.py

```diff
@@ -2822,7 +2831,7 @@ def _visit_Return(self, expr, **settings):
 
     def _visit_FunctionDef(self, expr, **settings):
 
-        name            = expr.name
+        name            = self.scope.get_expected_name(expr.name)
         name            = name.replace("'", '')
```

Comment:
Is this replace operation still relevant? If yes, could it cause issues when calling
```python
python_name = expr.scope.get_python_name(name)
```
later on?

Reply:
Hmm, yes I think it could cause issues. But I don't think it is relevant. I assume it was created to handle a quoted FunctionDef name (which we shouldn't have). I have removed it 3a884774 so we shall see